lib/facade_converter.ts

```diff
@@ -91,7 +91,7 @@ export class FacadeConverter extends base.TranspilerBase {
 
     var fileName = symbol.valueDeclaration.getSourceFile().fileName;
     fileName = this.getRelativeFileName(fileName);
-    fileName = fileName.replace(/(\.d)?\.ts$/, '');
+    fileName = fileName.replace(/(\.d)?\.ts$/, '').replace(/\\/g, '/');
```

Comment:
Shouldn't `getSourceFile().fileName` already only contain `/` from the call to `createSourceFile` in `main.ts`?

Sprinkling the `.replace` all over the place will probably just make the problem worse. We should make sure that all paths that flow through the system use `/`.